docs/dev/proj.md

```diff
@@ -50,6 +50,34 @@ pipx install copier
     SKIP=no-commit-to-branch git push
     ```
 
+1. Bootstrap the dev container.
+
+    ::::{tab-set}
+
+    :::{tab-item} GitHub
+    :sync: github
+    Open the [Dev Container Workflow](https://github.com/serious-scaffold/ss-python/actions/workflows/devcontainer.yml) page and run the workflow from the default branch.
+
+    ```{image} /_static/images/bootstrap-dev-container-github.png
+    :alt: Bootstrap Dev Container on GitHub.
+    :target: https://github.com/serious-scaffold/ss-python/actions/workflows/devcontainer.yml
+    ```
+
+    :::
+
+    :::{tab-item} GitLab
+    :sync: gitlab
+    Open the [Run Pipelines](https://gitlab.com/serious-scaffold/ss-python/-/pipelines/new) page and run pipeline for the default branch.
+
```

Comment:
maybe pipelines😅

Reply:
Done, according to the screenshot, I changed the first appearance to "Run pipeline". Let me know if there is any misunderstanding.